[WIP] Election 2020 (#1216)
arviz-devs/arviz: elections/ArviZ_2020.md

# Election 2020

## Summary of Process
This is the first election cycle for ArviZ. During this cycle we are aiming
for the following outcomes

* Ratification of Governance.MD
* Identification and listing of Core Contributors
* Nominations for Council Candidates 
* Council Elections. Particularly
  * Size of Council

>   * Inaugral Arviz Council

Reviewer: ```suggestion
  * Inaugural ArviZ Council
```

## Process Details
### Identification of Core Contributors (1 week)

> 1. Google Form will be created to get core contributor names and emails

Reviewer: ```suggestion
1. A Google Form will be created to get core contributor names and emails
```

    * Emails are needed for Elections steps so NumFOCUS can mail and track votes

>     * Google Form is moderated by Ravin

Reviewer: ```suggestion
    * The Google Form is moderated by Ravin
```

> 2. At conclusion of week Ravin will update PR with all names and make forms responses public,

Reviewer: ```suggestion
2. At conclusion of week Ravin will update PR with all names and make the form's responses public,
```

with exception of email

### Nominations for Council (1 week)
1. A github issue labeled *Arviz 2020 Council Elections* will be opened for nominations

> 2. Anyone can make a nominate a core contributor (including general public)

Reviewer 2: Typo: "can ~~make a~~ nominate a core contributor"

Author: Accepting all of these but quick tip for you. You can suggest edits directly in github, that can then be committed straight from the browser, that way  they dont need to be typed twice :)

Reviewer 2: Thanks, didn't know about that! Will look for this functionality next time then ;)

> 3. Core Contributor must accept acknowledgement with thumbs up on nomination

Reviewer 2: If he doesn't, this means his name isn't considered for Council Election, right? Maybe add this precision?

Author: Their name isn't considered because we don't know if they willingness or time to exercise the role

Author: I added a note for that

> 4. At end week all nominations will be compiled in last coment 

Reviewer 2: "At the end of the week, ..."

### First Round Election for Council and Council size (1 week)
The first election will be for council and council size. Elections will take place over 

> a Google Forums owned by NumFOCUS. The tallying will be performed in accordance with `GOVERNANCE.md`

Reviewer: ```suggestion
a Google Form owned by NumFOCUS. The tallying will be performed in accordance with `GOVERNANCE.md`
```

1. NumFOCUS will email all registered voters a ballot for a vote on
    * Council Size
    * Council Members
   
 
### Tie Votes (1 week)

> In the event of a tie there will be a run off election. 

Reviewer: ```suggestion
In the event of a tie there will be a run-off election. 
```

# Election 2020

## Summary of Process
This is the first election cycle for ArviZ. During this cycle we are aiming
for the following outcomes

* Ratification of Governance.MD
* Identification and listing of Core Contributors
* Nominations for Council Candidates 
* Council Elections. Particularly
  * Size of Council
  * Inaugural ArviZ Council

## Process Details
### Identification of Core Contributors (1 week)
1. A Google Form will be created to get core contributor names and emails
    * Emails are needed for Elections steps so NumFOCUS can mail and track votes
    * The Google Form is moderated by Ravin
2. At conclusion of week Ravin will update PR with all names and make the form's responses public,
with exception of email

### Nominations for Council (1 week)
1. A github issue labeled *Arviz 2020 Council Elections* will be opened for nominations

> 2. Anyone can nominate a core contributor (including general public)

Reviewer 3: The 1 week term and general public being able to nominate diverges from what is said in governance. We should probably have a couple lines explaining that due to this being the first there will be some exceptions (these are not the only ones, we'll also be voting for number of council members for example)

Author: Fair point, let me edit Governance to say something less than no less than 1 week or as not to be too restrictive

# Election 2020

## Summary of Process
This is the first election cycle for ArviZ. During this cycle we are aiming
for the following outcomes

* Ratification of GOVERNANCE.md
* Identification and listing of Core Contributors
* Nominations for Council Candidates
* Council Elections. Particularly
  * Size of Council
  * Inaugural ArviZ Council

This being the first election means that the process will have more steps than
the ones listed in GOVERNANCE.md.

> The moderators for these tasks live in the PST and CST timezone so the dates are reflective of their localities.

Reviewer 4: What does this mean? What timezone is used?

Can we have utc times also visible?

Author: It basically means when I say a date it's that date in the American timezones. So I might do something June 8th at 8 pm my time after work, but for you guys it will already be June 9th

### Nominations for Council (1 week)
1. A github issue labeled *Arviz 2020 Council Elections* will be opened for nominations
2. Anyone can nominate a core contributor (including general public)
3. Core Contributor must accept acknowledgement with thumbs up on nomination

>    * Without an acknowledgement they will not be a candidate ni the election

Reviewer 2: ```suggestion
   * Without an acknowledgement, they will not be a candidate in the election.
```

   * A nomination can be explicitly declined as well in a comment, if that individual

>    feels they will not be able serve as a Council Member for any reason

Reviewer 2: ```suggestion
   feels they will not be able to serve as a Council Member for any reason.
```

> 4. At the end week all nominations will be compiled in last comment

Reviewer 2: ```suggestion
4. At the end of the week, all nominations will be compiled in last comment.
```
